tests/ssr.test.tsx

```diff
@@ -11,10 +11,18 @@ jest.mock('react', () => {
 
 describe('Static-Style-Extract.SSR', () => {
   it('not warning', () => {
+    const allowedWarnings = [
+      'Warning: [antd: List] The `List` component is deprecated. And will be removed in next major version.'
+    ]
```

Comment:
@afc163 this test was failing due to this deprecation warning. i'm assuming this test was meant to catch extraction errors and that this deprecation warning can be ignored so I updated the test logic accordingly.